docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>"To find the top discord"</p><p> </p><p>"one can compute the left matrix profile and..." (i.e., omit "array")</p><p> </p><p>"approximate" (do not truncate)</p><p> </p><p>"Let's say we have a time series T and we begin checking each subsequence starting from left to right and updating our best-so-far discord as a rarer, more unique subsequence is encountered. At the i-th subsequence, the distance for the best-so-far discord is simply the distance between the rarest/most unique subsequence discovered somewhere amongst <code>T[:i]</code> and its nearest neighbor."</p><p> </p><p>I think we should just call it "discord distance" or "just distance" rather than "discord score". At the end of the day, it's just a Euclidean distance, right?</p><p> </p><p>"Naively, one can compare all of the subsequences, <code>T[:i]</code>,that exist to the left of i-th subsequence, <code>S_i = T[i : i + m],</code> and then find its one-nearest-neighbor. However, this is very time consuming and DAMP employs an early abandoning approach that helps to dramatically reduce this computational time." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks for your several suggestions and feedback! I will revise the notebook accordingly.</p>